src/common/tracing.ts

```diff
@@ -7,4 +8,5 @@ export const tracing = new Tracing([
     ignoreIncomingPaths: IGNORED_INCOMING_TRACE_ROUTES,
     ignoreOutgoingUrls: IGNORED_OUTGOING_TRACE_ROUTES,
   }),
+  new ExpressInstrumentation(),
```

Comment:
Where did you get that?

Reply:
because the upgrade to v3

Reply:
It is not related to S3. It is taken from boilerplate. Keep it

Reply:
ok